06-reproj.Rmd

```diff
@@ -135,7 +135,7 @@ There are no clear-cut answers to these questions and CRS selection always invol
 However, there are some general principles provided in this section that can help you decide. 
 
 First it's worth considering *when to transform*.
-In some cases transformation to a projected CRS is essential, such as when using geometric functions such as `st_buffer()`, as Figure \@ref(fig:crs-buf) shows.
+In some cases transformation to a projected CRS is essential, such as when using geometric functions such as `st_buffer()`, as Figure \@ref(fig:crs-buf) showed.
```

Comment:
It still shows it, no? I think present tense is more appropriate here. But happy to hear arguments in favour of using past tense.

Reply:
True. I think that both are correct. The figure still shows it, so you can keep the present if you interpret it in that way. If you interpret it in the way of "as we already saw in the figure xx" (as I did), then the preterite is more consistent with the rest of the book since you usually refer to things already covered in the preterite (e.g. `Like the st_transform() function demonstrated in the previous section (...)`, `Section 2.4 introduced coordinate reference systems (CRSs) and demonstrated their importance.`, `Chapter 2 demonstrated how vector geometries are made-up of points (...)`, etc.)

So I guess it is up to you!

Reply:
You can consider that the figure shows it (atemporel reality---this is always true), or the figure showed it in the previous paragraph/section/whatever (temporality of the reader going through the book).

Reply:
Stylistically, I prefer the preterite. But I am happy to revert to the present. You are the author :slightly_smiling_face:

Reply:
To me this is more consistent with the rest because when you wrote `Chapter 2 demonstrated how vector geometries are made-up of points (...)`, one could argue that Chapter 2 still demonstrates how... etc.

Reply:
So, usually, you do place yourself in the temporality of the reader going through the book. Which is why this one time you did not struck me.

Reply:
Good reasoning. Happy with that.